clodius/cli/convert.py

```diff
@@ -39,9 +39,40 @@ def epilogos_bedline_to_vector(bedline):
     chrom=parts[0]
     start=int(parts[1])
     end=int(parts[2])
-    
+
     return (chrom, start, end, states)
 
+def states_bedline_to_vector(bedline,states_dic):
+    '''
+    Convert a line from a bedfile containing states in categorical data to vector format.
+
+    Parameters
+    ----------
+
+    parts: [string,...]
+        A line form a bedfile broken up into its contituent parts
+        (e.g. ["chr1", "1000", "2000", "state"]))
+
+
+    states_dic:
+        A dictionary containing the states in the file with a corresponding value
+
+    Returns
+    -------
+
+    An array containing values associated the state
+
+    '''
+    parts = bedline.decode('utf8').strip().split('\t')
```

Comment:
if `bedline` is an array as it says in the parameter definition above, the `.decode` function shouldn't work. I'm guessing that `bedline` is actually a string?

Reply:
Yeah,  bedline is just the line from the bedfile.
I meant that the function returns at the end an array of values associated with that line like chrom, start, end, states_vector. I will give a more specific example.